src/main/java/com/amazon/kinesis/kafka/IAMUtility.java

```diff
@@ -0,0 +1,29 @@
+package com.amazon.kinesis.kafka;
```

Comment:
This project is under Apache License Version 2.0. Could you please add the license header to the file?
Also, please add Javadocs to the class and a method.